ansible/roles/ocp-workload-velero/templates/restic-daemonset.yml.j2

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata: 
+  name: restic
+  namespace: velero
+spec:
+  selector:
+    matchLabels:
+      name: restic
+  template:
+    metadata:
+      labels:
+        name: restic
+    spec:
+      serviceAccountName: velero
+      securityContext:
+        runAsUser: 0
+      volumes:
+        - name: {{ velero_aws_secret_name }}
+          secret:
+            secretName: {{ velero_aws_secret_name }}
+        - name: host-pods
+          hostPath:
+            path: /var/lib/kubelet/pods
+        - name: scratch
+          emptyDir: {}
+      containers:
+        - name: velero
+          image: quay.io/ocpmigrate/velero:fusor-dev
```

Comment:
+1 on having this use the images we've patched.

I think it'd be worth making this a variable in defaults that can be overridden